cluster/get-kube.sh

```diff
@@ -162,8 +178,20 @@ case "${machine}" in
 esac
 
 file=kubernetes.tar.gz
-release=${KUBERNETES_RELEASE:-$(get_latest_version_number)}
-release_url="${KUBERNETES_RELEASE_URL}/${release}/${file}"
+release=${KUBERNETES_RELEASE:-"release/stable"}
+
+# Validate Kubernetes release version.
+# Translate a published version <bucket>/<version> (e.g. "release/stable") to version number.
+set_binary_version "${release}"
```

Comment:
there are a few spots later in this file that probably need to use `${KUBE_VERSION}` instead of `${release}`, but GitHub won't let me leave a comment there.